Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.16
+
+WORKDIR /app
+
+COPY . .
+
+RUN go build ./cryptgo.go
+
+CMD ["./cryptgo"]
```

Comment:
@matheusmosca it would probably better to use `ENTRYPOINT` rather than `CMD` as that would allow users to use the flags associated with the binary too.